TWLight/users/management/commands/user_update_eligibility.py

```diff
@@ -40,7 +41,12 @@ def handle(self, *args, **options):
         if options["datetime"]:
             wp_editcount_updated = datetime.fromisoformat(options["datetime"])
 
-        editors = Editor.objects.filter(wp_bundle_eligible=True)
+        # Getting all editors that are currently eligible or are staff or are superusers
```

Comment:
I would revert this change and use a simple migration to set all staff + superusers to eligible.

Reply:
Got it! I am removing some test cases I added, since this will break them (but I don't think we'll have an actual problem in production because of the migration I created.